config.json

```diff
@@ -3,7 +3,7 @@
   "slug": "haskell",
   "active": true,
   "status": {
-    "concept_exercises": false,
+    "concept_exercises": true,
```

Comment:
I'm not sure we should set this to true if there is only one concept exercise. @iHiD?

Reply:
No, not until there's a quorum of exercises.